<ui version="4.0" >
 <author>Kevin Ottens &lt;ervin@kde.org></author>
 <class>BackendChooser</class>
 <widget class="QWidget" name="BackendChooser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>376</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="listView" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="dragEnabled" >
           <bool>false</bool>
          </property>
          <property name="dragDropMode" >
           <enum>QAbstractItemView::NoDragDrop</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="upButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>prefer the selected backend</string>
            </property>
            <property name="text" >
             <string>move up</string>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
            <property name="arrowType" >
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="downButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>no preference for the selected backend</string>
            </property>
            <property name="text" >
             <string>move down</string>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
            <property name="arrowType" >
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="font" >
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="iconLabel" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>0</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nameLabel" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>5</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="alignment" >
              <set>Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="versionLabel" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>5</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="alignment" >
              <set>Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line1" >
           <property name="frameShape" >
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commentLabel" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="scaledContents" >
            <bool>false</bool>
           </property>
           <property name="alignment" >
            <set>Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
